pebblo/entity_classifier/utils/config.py

```diff
@@ -4,6 +4,17 @@
 from enum import Enum
 
 
+secret_entities_context_mapping = {
+    "Github Token": ["github", "github_token"],
```

Comment:
Is it possible to use regex here, instead of exact strings? That would give more coverage for context.

Reply:
@shreyas-damle Here we need to give exact strings only, we can give all possible assumptions here